File: conditionals/time_of_day.c
// Governor-Nephi Bakare, Time of Day C
#include <stdio.h>
#include <time.h>



int main(void){
    time_t now = time(NULL);
    struct tm * tm_struct = localtime(&now);
    int ahora = tm_struct->tm_hour;
    printf("%d\n", ahora);

    if (ahora <= 12){
        printf("TOP OF THE MORNING TO YOU!");
    }else if (ahora <= 16){
        printf("Good afternoon, user!");
    }else{
        printf("Good evening, user!");
    }
    return 0;
}

File: conditionals/time_of_day.py
# Governor-Nephi Bakare, Time of Day Python
import time


current = time.time()

local_time= time.localtime(current)
ahora = local_time.tm_hour

if ahora <= 12:
    print("Good morning, user! 🔥")
elif ahora <= 16:
    print("Good afternoon, user! 👍")
else:
    print("Good evening, user! 😃")